Automattic/jetpack
# Turn off css tidy shorthand optimisation

Fixes # https://github.com/Automattic/wp-calypso/issues/52310 & https://github.com/Automattic/jetpack/issues/19224

#### Changes proposed in this Pull Request:

* Turns off the css shorthand optimisation option. More blocks are now making use of inlines styles to add extra padding and margin options, and if the jetpack css optimiser reduces these to a single selector then the block invalidates. It may be possible to get a change into gutenburg that makes the blocks aware of this optimisation but that may take a while to get tested and approved so this change will likely be needed in the interim anyway.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

- On WPcom site post add a group block within another group block
- Set the padding of the child group in the left block settings panel
- Save the post and reload the editor and notice the child block is invalid
- Apply this patch and follow steps above to add a new group block, and this time when reloading child block should be valid
- On a WPcom site also add some longhand selectors via Customizer additional css and check that these are still converted to shorthand in the frontend
- Also test an Atomic site - seems like they are not affected by this css tidy code as couldn't replicate the validation issue, but worth somebody double checking

Before:
<img width="973" alt="Screen Shot 2021-05-04 at 3 34 44 PM" src="https://user-images.githubusercontent.com/3629020/116958313-484afd00-acee-11eb-857a-af2a8cd16d10.png">

After:

<img width="975" alt="Screen Shot 2021-05-04 at 3 38 05 PM" src="https://user-images.githubusercontent.com/3629020/116958434-bf809100-acee-11eb-970a-75defb1502dd.png">
